### PR TITLE
switch to outline-ss-server releases

### PR DESCRIPTION
**Pull request by trevj**
I'll create 0.0.1 once https://github.com/Jigsaw-Code/outline-ss-server/pull/4 is merged.

### CODE REVIEW

File: src/shadowbox/docker/Dockerfile [hunk 1]
```diff
@@ -12,16 +12,15 @@
...
-RUN go get github.com/Jigsaw-Code/outline-ss-server
-
 # See versions at https://hub.docker.com/_/node/
 FROM node:8.11.3-alpine
 
 # Versions can be found at https://github.com/shadowsocks/shadowsocks-libev/releases
 ARG SS_VERSION=3.2.0
 
+# Versions can be found at https://github.com/Jigsaw-Code/outline-ss-server/releases
+ARG SS_GO_VERSION=0.0.1
```
Comment (fortuna) [inline src/shadowbox/docker/Dockerfile]: We should start with 1.0.0. Now that we are using it, we can't break compatibility anymore.

Comment (trevj) [reply src/shadowbox/docker/Dockerfile]: Sure, done.

**Comment by trevj** [comment]
@fortuna CI is passing now that https://github.com/Jigsaw-Code/outline-ss-server/releases/tag/v1.0.0 exists, PTAL :-)

File: src/shadowbox/docker/Dockerfile [hunk 2]
```diff
@@ -12,16 +12,15 @@
...
 
 # Versions can be found at https://github.com/shadowsocks/shadowsocks-libev/releases
 ARG SS_VERSION=3.2.0
 
+# Versions can be found at https://github.com/Jigsaw-Code/outline-ss-server/releases
+ARG SS_GO_VERSION=1.0.0
+
 # Save metadata on the software versions we are using.
 LABEL shadowbox.node_version=8.11.3
 LABEL shadowbox.shadowsocks_version="${SS_VERSION}"
```
[the remark this answers is not in the document]
Comment (trevj) [reply src/shadowbox/docker/Dockerfile]: I think we should change the name. I just tried building with `shadowbox.outline-ss-server_version` and Docker didn't complain about the hyphen.

[the remark this answers is not in the document]
Comment (fortuna) [reply src/shadowbox/docker/Dockerfile]: Sounds good

**Pull request merged**